.travis.yml

```diff
@@ -0,0 +1,52 @@
+language: node_js
+node_js:
+- '6'
+cache:
+  directories:
+  - node_modules
+install: npm install
+script: npm test
+before_deploy: echo 'All unit tests passed;
+  Preparing to deploy Get a Library Card to AWS'
+deploy:
+- provider: elasticbeanstalk
+  skip_cleanup: true
+  access_key_id: "$AWS_ACCESS_KEY_ID_PRODUCTION"
+  secret_access_key: "$AWS_SECRET_ACCESS_KEY_PRODUCTION"
+  region: us-east-1
+  app: nypl-library-card-app
+  env: nypl-library-card-app-production
+  bucket_name: elasticbeanstalk-us-east-1-946183545209
+  bucket_path: nypl-library-card-app-production
+  on:
+    repo: NYPL/nypl-library-card-app
+    branch: production
+- provider: elasticbeanstalk
+  skip_cleanup: true
+  access_key_id: "$AWS_ACCESS_KEY_ID_PRODUCTION"
+  secret_access_key: "$AWS_SECRET_ACCESS_KEY_PRODUCTION"
+  region: us-east-1
+  app: nypl-library-card-app
+  env: nypl-library-card-app-qa
+  bucket_name: elasticbeanstalk-us-east-1-946183545209
+  bucket_path: nypl-library-card-app-qa
+  on:
+    repo: NYPL/nypl-library-card-app
+    branch: qa
+- provider: elasticbeanstalk
+  skip_cleanup: true
+  access_key_id: "$AWS_ACCESS_KEY_ID_DEVELOPMENT"
+  secret_access_key: "$AWS_SECRET_ACCESS_KEY_DEVELOPMENT"
+  region: us-east-1
+  app: nypl-library-card-app
+  env: nypl-library-card-app-development
+  bucket_name: elasticbeanstalk-us-east-1-224280085904
+  bucket_path: nypl-library-card-app-development
+  on:
+    repo: NYPL/nypl-library-card-app
+    branch: development
```

Comment:
For library card, the  branch for development env should be `master`, and for production env, it is `production`